docs/guides/setup/installing/local.md

```diff
@@ -1,5 +1,89 @@
 ---
-title: local python
+title: Installing in great_expectations in Local Python3
```

Comment:
Please make titles consistent with the specific language in the design document: https://github.com/superconductive/design/blob/main/docs/20210421_knoxpod_golden_paths_mental_map/toc.md#installation-this-still-needs-to-be-fleshed-out-and-finalized

Reply:
Done